### PR TITLE
Update #1790 (Python 3 fixes)

### PR DESCRIPTION
**Pull request by ericholscher**
This removes the dependency on six (it was only used once),
but keeps all the proper exception handling.

Refs #1790

### CODE REVIEW

File: readthedocs/oauth/migrations/0003_move_github.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def forwards_move_repos(apps, schema_editor):
             else:
                 new_repo.clone_url = data.get('clone_url')
             new_repo.json = json.dumps(data)
-        except SyntaxError, ValueError:
+        except SyntaxError as ValueError:
```
Comment (agjohnson) [inline readthedocs/oauth/migrations/0003_move_github.py]: Looks like a find replace failure

File: readthedocs/oauth/migrations/0003_move_github.py [hunk 2]
```diff
@@ -149,7 +149,7 @@ def forwards_move_repos(apps, schema_editor):
                 new_repo.clone_url = clone_urls.get('ssh', project.git_url)
             else:
                 new_repo.clone_url = clone_urls.get('https', project.html_url)
-        except SyntaxError, ValueError:
+        except SyntaxError as ValueError:
```
Comment (agjohnson) [inline readthedocs/oauth/migrations/0003_move_github.py]: Same here.

**Comment by ericholscher** [comment]
Fixed ^^

**Comment by agjohnson** [comment]
:+1:

**Comment by ivoflipse** [comment]
Still getting these docutils problems on RTD.com

**Pull request merged**